locales/en/breaches.ftl

```diff
@@ -136,7 +136,7 @@ breach-checklist-phone-header-2 = Protect your phone number with a masking servi
 ## Prompts the user for changes when there is a breach detected of security questions
 
 # { $breachedCompanyLink } will link to the website of the company where the breach occurred
-breach-checklist-sq-header-2 = Update your security questions on the company’s website.
+breach-checklist-sq-header-3 = Update your security questions on { $breachedCompanyLink }.
```

Comment:
I don't understand these changes. I thought we agreed a couple of weeks ago that we don't want links?
https://github.com/mozilla/blurts-server/pull/2900#discussion_r1135799859

Reply:
Yes, there was some back and forth. Ultimately PM decided to add those links and [MNTOR-1504](https://mozilla-hub.atlassian.net/browse/MNTOR-1504) was created.

Reply:
cc @EMMLynch